readme.md

```diff
@@ -91,32 +94,39 @@ The second part is done in Bitbucket Server and involves creating an Application
 <img src="images/linkapplications.png" width="600"> <br/>
 
 To create the Application Link: 
-1. In Bitbucket Server go to **Administration** > **Application Links**. 
+1. In Bitbucket Server, go to **Administration** > **Application Links**. 
 2. Enter the application URL (see Application Link details page) and select **Create new link**. 
 3. Select **Continue** on the warning message. This is not a problem. 
-4. Complete all the fields in the form (see Application Link details page for some fields). 
-5. Leave **Create incoming link** unchecked. 
-6. Select **Continue**. 
+4. Complete the form: 
+   - Application Name - Enter a name for your Jenkins instance. 
+   - Application Type - Leave as Generic Application.
+   - Service Provider Name - Enter ??? Or same as description??? 
+   - Consumer Key - See Application Link details page. 
+   - Shared secret - Enter the secret you created when registering Bitbucket Server as a consumer. 
+   - Request Token URL - See Application Link details page. 
+   - Access token URL - See Application Link details page. 
+   - Authorize URL - See Application Link details page. 
+   - Create incoming link - Leave unchecked. 
+5. Select **Continue**. 
 
 After a moment, your Jenkins instance will appear in the list of linked applications. 
 
 ---
 
 ## Use the plugin 
 
-### Create a job
+### Select a Bitbucket Server instance when creating a freestyle job
```

Comment:
Maybe capitalize Freestyle Job (to make consistent with the Create Multibranch below)

Reply:
```suggestion
### Select a Bitbucket Server instance when creating a Freestyle Job
```